database/tables/stripe_checkout_sessions.pg

```diff
@@ -0,0 +1,19 @@
+columns
+    completed_at: timestamp with time zone
+    course_instance_id: bigint
+    created_at: timestamp with time zone not null default now()
+    data: jsonb not null
+    id: bigint not null default nextval('stripe_checkout_sessions_id_seq'::regclass)
+    plan_grants_created: boolean not null default false
+    plan_names: text[] not null
+    stripe_object_id: text not null
+    user_id: bigint not null
```

Comment:
I'm inclined to say this table should track `authn_user_id` separately from `user_id` so that we can in theory support a user buying a plan grant for an entire institution. `authn_user_id` can track who actually made the purchase; `user_id` can track which user (if any) the purchase was made for. For now, these will be the same.

Reply:
Agreed. Maybe it's worth calling these something like `purchasing_user_id` and `subject_user_id` to clearly differentiate them (please bikeshed those names). I fear that using `authn_user_id` and `user_id` as column names is very unclear, given that we aren't using them in the usual sense of "true user id" versus "emulated user id".

Reply:
True, maybe `subject_user_id` and `agent_user_id` to match the terminology we've been using in PT?

Reply:
I switched to this terminology in 7198cca.

Reply:
I like the use of the PT terminology here.